webpack.config.js

```diff
@@ -45,7 +45,7 @@ function getDependeciesFromNpm(mod) {
 /**@type {import('webpack').Configuration}*/
 const config = {
     target: 'node',
-
+    mode: "development",
```

Comment:
It is better not to hardcode mode in webpack config, since webpack support both development and production mode and we need to use production mode to build package.